database/migrations/2015_11_24_160035_create_team_keys_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateTeamKeysTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('team_keys', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('public_key', 2024);
+            $table->string('private_key', 2024);
```

Comment:
Shouldn't we go for a length of 2048 rather than 2024? http://danielpocock.com/rsa-key-sizes-2048-or-4096-bits